ffs/scheduler/scheduler_storage.go

```diff
@@ -105,6 +105,11 @@ func (s *Scheduler) GetStorageInfo(iid ffs.APIID, c cid.Cid) (ffs.StorageInfo, e
 	return info, nil
 }
 
+// QueryStorageInfo returns a list of StorageInfo matching any provided query options.
+func (s *Scheduler) QueryStorageInfo(iids []ffs.APIID, cids []cid.Cid) ([]ffs.StorageInfo, error) {
```

Comment:
Just bubbling the new `cistore` method up into the scheduler layer. This is used by the FFS API, and if the general api looks good, the `Admin` gRPC api will call this too.

Reply:
Nit: Mind wrapping the error to have some more path-explanation of it for this layer?